docs/sources/flow/reference/cli/run.md

```diff
@@ -31,6 +31,9 @@ The following flags are supported:
 * `--server.http.ui-path-prefix`: Base path where the UI will be exposed (default `/`).
 * `--storage.path`: Base directory where components can store data (default `data-agent/`).
 * `--disable-reporting`: Disable [usage reporting][] of enabled [components][] to Grafana (default `false`).
+* `--cluster.enabled`: Start the Agent in clustered mode (default `false`).
+* `--cluster.join-address`: Address to join the cluster at (default `""`).
```

Comment:
Should this be `--cluster.join-addresses`? To show that it can be multiple (or one)

Reply:
Yep, sounds better.